src/org/zaproxy/zap/extension/groovy/Messages.properties

```diff
@@ -0,0 +1,3 @@
+# This file defines the default (English) variants of all of the internationalised messages

```

Comment:
This file should be in the resources directory.

Reply:
It's also picked from this location (although considered an old location).